frontend/web/index.handlebars

```diff
@@ -9,11 +9,10 @@
     <meta property="og:image" content="https://flagsmith.com/static/aa08b77a1a9bca99643945f1f3eeb027/b2d24/social-preview.jpg" />
     <meta property="og:title" content="Flagsmith - Feature Flags, Feature Toggles and Remote Config" />
     <meta name="robots" content="noindex, nofollow"/>
-    <script src="/static/javascript/highlight.min.js"></script>
-    <script src="/static/javascript/msal.min.js"></script>
```

Comment:
As mentioned in the description, I'm not sure if we should / can be removing the msal dependency here, but I can't see any reason that it exists. Maybe for LDAP authentication I guess? 

@kyle-ssg probably knows.